rct229/rules/section5.py

```diff
@@ -0,0 +1,58 @@
+from numpy import sum
+
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.jsonpath_utils import find_all
+
+# Rule Definitions for Section 5 of 90.1-2019 Appendix G
+
+
+# ------------------------
+
+
+class Section5Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
```

Comment:
I maybe wrong, the class name suggests Section 5 Rule 3, but the description/documentation mentions rule 6. Please ignore otherwise.

Reply:
Good catch.  I will change that.